doc/weakly_supervised.rst

```diff
@@ -0,0 +1,316 @@
+=================================
+Weakly Supervised Metric Learning
+=================================
+
+Weakly supervised algorithms work on weaker information about the data points
+than supervised algorithms. Rather than labeled points, they take as input
+similarity judgments on tuples of data points, for instance pairs of similar
+and dissimilar points. Refer to the documentation of each algorithm for its
+particular form of input data.
+
+
+Input data
+==========
+
+In the following paragraph we talk about tuples for sake of generality.
+These can be pairs, triplets, quadruplets etc, depending on what algorithm we
```

Comment:
depending on the particular metric learning algorithm we use.

Reply:
That's right, done